src/database/migrations/1716831066683-AddTimezoneForBlockedIps.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
please use migration api https://orkhan.gitbook.io/typeorm/docs/migrations await queryRunner.addColumn ...

Reply:
If user queryRunner.changeColumn it does not update the column but drops it and creates it again

Reply:
And it's possible data loss